ui/components/multichain/connections-header/connections-header.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { ConnectionsHeader } from './connections-header';
+
+export default {
+  title: 'Components/Multichain/ConnectionsHeader',
+};
+export const DefaultStory = () => <ConnectionsHeader />;
+
+DefaultStory.storyName = 'Default';
+
+export const ChaosStory = () => (
+  <div style={{ width: '400px' }}>
+    <ConnectionsHeader hostName="some.weird.hostname.weve.never.heard.of.tld" />
+  </div>
+);
+
+ChaosStory.storyName = 'Chaos';
```

Comment:
Here's a way that will make the story interactive and you get some small performance benefits for stoybook by using bind

```
import React from 'react';
import { ConnectionsHeader } from './connections-header';

export default {
  title: 'Components/Multichain/ConnectionsHeader',
  argType: {
    hostName: { control: 'text' },
  },
};

const Template = (args) => <ConnectionsHeader {...args} />;

export const DefaultStory = Template.bind({});
DefaultStory.storyName = 'Default';

export const Chaos = Template.bind({});
Chaos.args = {
  hostName: 'some.weird.hostname.weve.never.heard.of.tld',
};
```

Reply:
For the Chaos Story, we use 400px width to trigger the overflow and ellipsis of the siteName. Would it be possible to keep that behavior for the Chaos story while also using Template.bind? @georgewrmarshall

Reply:
I think you could wrap it using a provider but if that's the case what you have is fine